core/capabilities/ccip/ccipevm/pluginconfig.go

```diff
@@ -21,3 +21,23 @@ func InitializePluginConfig(lggr logger.Logger, extraDataCodec ccipcommon.ExtraD
 		ContractTransmitterFactory: ocrimpls.NewEVMContractTransmitterFactory(extraDataCodec),
 	}
 }
+
+func init() {
```

Comment:
Is there a way to register the entire Plugin config object instead of each of these individually?